app/utils/error_handlers.py

```diff
@@ -3,66 +3,109 @@
 
 class DatabaseConnectionError(HTTPException):
     """handle database connection error"""
-    code = 500
-    description = "An error occurred while connecting to the database"
+    status = 500
 
 
 class TableCreationError(HTTPException):
     """handle table creation error"""
-    code = 500
-    description = "An error occurred while creating the tables"
 
 
 class InvalidEmailFormatError(HTTPException):
     """handle invalid email format"""
-    code = 400
-    description = "Invalid email format"
 
 
 class UserAlreadyExistsError(HTTPException):
     """handle duplicate user registration"""
-    code = 409
-    description = "A user with that email already exists"
 
 
 class InvalidPasswordLengthError(HTTPException):
     """handle duplicate user registration"""
-    code = 400
-    description = "Password length has to be at least 6 characters"
 
 
 class UserLoginError(HTTPException):
     """handle invalid login details"""
-    code = 401
-    description = "Invalid login details provided"
 
 
 class DuplicateDataError(HTTPException):
     """handle duplicate data entries"""
```

Comment:
Why is it that you have left all the Classes in `app/utils/error_handlers.py` empty?

Reply:
The classes are not really empty per se. Please check the `errors = ... ` at the bottom of the file where each class is called. It's the same approach that is taken in the `config.py` only a bit different. This is the way to do it according to [flask-restful docs](https://flask-restful.readthedocs.io/en/0.3.5/extending.html#define-custom-error-messages).

Reply:
It's okay, though in that documentation I don't see an area where there are classes created and left empty. You will enlighten me more on it tomorrow.

Reply:
That's ok. We can discuss more on it tomorrow. The documentation is scanty. In the meantime, you can have a look at [this](https://stackoverflow.com/a/32449070/9312256) thread.